github-actions/commit-message-based-labels/lib/main.ts

```diff
@@ -46,6 +48,13 @@ class CommitMessageBasedLabelManager {
         await this.removeLabel(name);
       }
     }
+
+    for (const commit of this.commits) {
+      const label = 'area: ' + commit.scope;
+      if (this.repoAreaLabels.has(label) && !this.labels.has(label)) {
```

Comment:
We don't need to check if the repo as the area labels as we already know it will since its a managed label.

Reply:
What about the case with an invalid commit header? The lint check would fail, but then this action would run and add an invalid area label if we don't check. Someone could be like `fix(blah): stuff` and then this would just shove `area: blah` even though that's not what we want.

Reply:
AHh that makes sense, can you add commenting to express that for the future as well?

Reply:
Comment added!